templates/user/user.html

```diff
@@ -27,6 +27,8 @@
     <div>Github: <a href="https://github.com/{{ user.handle }}" target="_blank">{{ user.handle }}</a></div>
     <div>Mattermost: {{ user.mmost }}</div>
     <div>Member since: {{ user.joined_on }}</div>
+    <div>Former Member?: {{ user.former }}</div>
+    <div>Working on external project?: {{ user.extrn }}</div>
```

Comment:
The external field actually means that this is someone who is not an RCOS member who has registered on Observatory. The external flag on projects #21 is different.